Cloud Application Manager/Deploying Anywhere/using-your-aws-account.md

```diff
@@ -490,6 +490,40 @@ When deploying via AWS, we register the instance to the load balancer and automa
 
 **Note:** Since you more frequently update or replace applications than load balancers, we recommend you reuse existing load balancers in production environments. This will help retain DNS settings that forward traffic to the instance.
 
+#### Deployment Information
+
+Once your instance has been deployed, you can access its information by clicking on it in the instances list view and accessing the instance details page. The right side of the screen displays instance deployment information. _Not all the attributes apply to every instance so some of them might not appear_:
+
+|         Concept      | Description |
+|----------------------|-------------|
+|Support ID            | This ID relates this instance in case you ask for support regarding it. There might be more than one IDs in case this CAM instance contains more than one vendor instances. |
```

Comment:
```suggestion
|Support ID            | This ID relates this instance in case you ask for support regarding it. There might be more than one ID in case this CAM instance is associated with more than one machine. |
```